packages/gatsby-core-utils/src/create-require-from-path.ts

```diff
@@ -0,0 +1,18 @@
+import Module from "module"
+import path from "path"
+
+const fallback = (filename: string): NodeRequire => {
+  // @ts-ignore
+  const mod = new Module(filename, null)
+
+  mod.filename = filename
+  // @ts-ignore
+  mod.paths = Module._nodeModulePaths(path.dirname(filename))
```

Comment:
```suggestion
/**
 * We need to use private Module methods in this polyfill
 */
interface IModulePrivateMethods {
  _nodeModulePaths: (directory: string) => string[]
  _compile: (src: string, file: string) => void
}

const fallback = (filename: string): NodeRequire => {
  const mod = new Module(filename) as Module & IModulePrivateMethods

  mod.filename = filename
  mod.paths = (Module as typeof Module &
    IModulePrivateMethods)._nodeModulePaths(path.dirname(filename))
```

I think it's better to explicitly type the private methods that we're using

Reply:
That was brilliant. Thanks Matt!